packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -361,15 +361,8 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     }
   }
 
-  a11yStatusTimeout: any
-  charKeysPressedTimeout: any
   defaultTriggerButtonId = _.uniqueId('dropdown-trigger-button-')
 
-  componentWillUnmount() {
-    clearTimeout(this.a11yStatusTimeout)
-    clearTimeout(this.charKeysPressedTimeout)
-  }
```

Comment:
Please add an `cancel()` call for `_.debounce`:

https://lodash.com/docs/4.17.15#debounce

Reply:
you mean `this.clearStartingString.cancel()` ?

Reply:
focusItemAtIndex.cancel()

Reply:
Solved. Thanks!